src/Component/Product/ProductInterface.php

```diff
@@ -14,7 +14,7 @@
 use Doctrine\Common\Collections\ArrayCollection;
 use Sonata\MediaBundle\Model\GalleryInterface;
 use Sonata\MediaBundle\Model\MediaInterface;
-use Symfony\Component\Validator\ExecutionContextInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
```

Comment:
You cannot change this, maybe just remove the type hint on the function using it?

Reply:
That will be incompatiable too. I'l remove this commit. Now I just need to fix travis build, because it installs symfony/validator 3.2 in one of the required builds.

Reply:
It is not BC break to remove a type hint, only to add it or change it from one to another

Reply:
Thanks, I'll do it in a separate PR later.